lighthouse-cli/test/smokehouse/test-definitions/dobetterweb/dbw-expectations.js

```diff
@@ -415,7 +415,7 @@ const expectations = [
             screenshot: {
               width: 360,
               // Allow for differences in platforms.
-              height: '3755±5',
+              height: '1350±20',
```

Comment:
the height of this page is now determined by text (which varies far more between OS) instead of a static height of a declared image, this distinction didn't seem so important as to add fixed heights to every text dom element.